packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-type/add-word-count.page.tsx

```diff
@@ -0,0 +1,121 @@
+import { GetServerSideProps, GetServerSidePropsContext } from 'next';
+import { getSessionIdFromCookies } from '../../../../../utils/session';
+import { getApplicationFormSummary } from '../../../../../services/ApplicationService';
+import QuestionPageGetServerSideProps from '../../../../../utils/QuestionPageGetServerSideProps';
+import Meta from '../../../../../components/layout/Meta';
+import CustomLink from '../../../../../components/custom-link/CustomLink';
+import { Button, FlexibleQuestionPageLayout, TextInput } from 'gap-web-ui';
+import { ApplicationFormSection } from '../../../../../types/ApplicationForm';
+import { postQuestion } from '../../../../../services/QuestionService';
+import { getSummaryFromSession } from '../../../../../services/SessionService';
+import { QuestionWithOptionsSummary } from '../../../../../types/QuestionSummary';
+import ResponseTypeEnum from '../../../../../enums/ResponseType';
+import InferProps from '../../../../../types/InferProps';
+
+type RequestBody = {
+  maxWords: string;
+};
+
+export const getServerSideProps = async (
+  context: GetServerSidePropsContext
+) => {
+  const { applicationId, sectionId } = context.params as Record<string, string>;
+  const sessionCookie = getSessionIdFromCookies(context.req);
+
+  const applicationFormSummary = await getApplicationFormSummary(
+    applicationId,
+    sessionCookie
+  );
+
+  const { sectionTitle } = applicationFormSummary.sections.find(
+    (section) => section.sectionId === sectionId
+  ) as ApplicationFormSection;
```

Comment:
These should live in fetchPageData so that errors are properly caught

Reply:
true, good spot :)